example/gluon/snail/README.md

```diff
@@ -0,0 +1,59 @@
+# SNAIL with Gluon
+
+---
+
+Gluon inplementation of [A Simple Neural Attentive Meta-Learniner](https://openreview.net/pdf?id=B1DmUzWAW)
+
+##### network structore
+![net_structure](assets/net_structure.png)
+
+##### building block structure
+![block_structure](assets/blocks.png)
+
+## Requirements
+- Python 3.6.1
+- Mxnet 1.2.0
+- pytorch 0.4.0 (only use data loading)
```

Comment:
Could you elaborate on that requirement? I can't find a code-reference to pytorch in your PR (besides the inspiration)

Reply:
Fix requirements
- delete pytorch
- add mxboard